.gitignore

```diff
@@ -20,6 +20,7 @@ m4/lt~obsolete.m4
 /json_config.h
 /compile
 /config.h
+/config.h.in
```

Comment:
my bad, I guess I should get another round of good sleep ;-) You are totally right!